src/TTextEdit.cpp

```diff
@@ -321,6 +330,28 @@ void TTextEdit::showNewLines()
         }
     }
     update();
+
+
+    if (QAccessible::isActive()) {
+        QString newLines = QString();
```

Comment:
It is unnecessary to assign a null `QString` the default constructor does that anyhow...
```suggestion
        QString newLines;
```

Reply:
Ah, nice to know! Thank you for the suggestion.

I modified the original commit making this change, so I didn’t use the “Commit suggestion” GitHub feature which would add a commit with your name to the branch.

If you prefer me to use the “Commit suggestion” feature, just let me know and I’ll be glad to do it.

Reply:
No, you don't have to use "Commit suggestion" - it is sometimes convenient but not always...